src/deploy/osp_deployer/checkpoints.py

```diff
@@ -160,7 +158,7 @@ def sah_health_check(self):
                 Ssh.execute_command(self.sah_ip,
                                     "root",
                                     self.settings.sah_node.root_password,
-                                    cmd)[1]:
+                                    cmd)[1].decode('utf-8'):
```

Comment:
Wouldn t it be more convenient to decode('utf-8') the string in the Sssh.execute_command rather than everytime we call it ?

Reply:
Will be updating the code for that. Thank you

Reply:
Updated